chainer_setup_build.py

```diff
@@ -297,7 +297,15 @@ def finalize_options(self):
 
             cython = get_cython_pkg()
             req_version = pkg_resources.parse_version(cython_version)
-            if cython is not None and cython.parsed_version > req_version:
+            if cython is None:
+                utils.print_warning('Cython is not installed')
+            elif cython.parsed_version < req_version:
+                utils.print_warning(
```

Comment:
This error message is confusing because a user can install chainer even in this case. How about printing that the installer only skips cythonizing?